tests/client/amo/components/TestLandingPage.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {
+  renderIntoDocument,
+  findRenderedComponentWithType,
+} from 'react-addons-test-utils';
+import { findDOMNode } from 'react-dom';
+import { Provider } from 'react-redux';
+
+import LandingPage from 'amo/components/LandingPage';
+import createStore from 'amo/store';
+import I18nProvider from 'core/i18n/Provider';
+import { getFakeI18nInst } from 'tests/client/helpers';
+
+
+describe('<LandingPage />', () => {
+  function render({ ...props }) {
+    const initialState = { api: { clientApp: 'android', lang: 'en-GB' } };
+
+    return findDOMNode(findRenderedComponentWithType(renderIntoDocument(
+      <Provider store={createStore(initialState)}>
+        <I18nProvider i18n={getFakeI18nInst()}>
+          <LandingPage {...props} />
```

Comment:
Because of all the HOC hotness, I think you can just do it like this now, right?

    <LandingPage store={createStore(initialState)} i18n={getFakeI18nInst()} {...props} />